coalib/output/printers/LogPrinterMixin.py

```diff
@@ -0,0 +1,45 @@
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
I don't see any other printers using it, or any other printer for that matter.
And in case it is being used, I think its better to shift it from everywhere.